BackEnd/Restaurant/Api/appsettings.json

```diff
@@ -7,7 +7,7 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DatabaseConnection": "Data Source=(localdb)\\local;Initial Catalog=Restaurant;Integrated Security=True"
+    "DatabaseConnection": "Data Source=(localdb)\\Local;Initial Catalog=Restaurant;Integrated Security=True"
```

Comment:
Are you sure about  this? why change this ?

Reply:
Because i have created new Database named Local instead of local